index.js

```diff
@@ -136,6 +140,36 @@ class Jarvis {
       }
     }
 
+    if (line.startsWith("in this context")) {
+      this.activeConstants = [];
+      const out = 'You are now entering constants. Type the constants, one line at a time. When done, type \'end\'.'
+      return out;
+    }
+
+    if (this.activeConstants) {
+      if (line === 'end') {
```

Comment:
shouldn't you trim this for trailing spaces?

Reply:
We didn't trim here because, for each `send` method invocation we trim the string at the top of the method.